modules/installing-ocp-agent.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * installing-with-agent/installing-with-agent.adoc
+
+:_content-type: PROCEDURE
+[id="installing-ocp-agent_{context}"]
+= Installing {product-title} with Agent
+WIP
+You can use the Agent-based installation to install a:
+
+* Single-node cluster
+* Compact cluster
+* Highly Available (HA) cluster
+
+The following procedure deploys a single-node {product-title} and you can use this procedure as a basis and modify according to your requirements.
+
+.Prerequisites
+* Golang version 1.17 and greater
+* RHEL version ?
+* 32GB memory
+* 120GB disk
+* 8 CPUs
+
+.Procedure
+
+. Navigate to the link:https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest-{product-version}/[OpenShift mirror site] and download the latest version of the tarball that matches your operating system.
+
+. Extract the binary.
++
+[source,terminal]
+----
+$ tar xvf <file> -C ~/.local/bin
+----
++
+The file contains the build for the **agent-installer** branch.
```

Comment:
I don't think we want to say "branch" here. It's in a branch today, but we expect to merge that into the mainline any day now.